src/components/secondPassphraseInput/secondPassphraseInput.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Input from 'react-toolbox/lib/input';
+import { isValidPassphrase } from '../../utils/passphrase';
+
+class SecondPassphraseInput extends React.Component {
+  componentDidMount() {
+    if (this.props.hasSecondPassphrase) {
+      this.props.onError(undefined, '');
+    }
+  }
+
+  handleChange(value) {
+    this.props.onChange(value);
+    const error = this.validate(value);
+    if (error) {
+      this.props.onError(error, value);
+    }
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  validate(value) {
```

Comment:
If you don't use 'this' in a function of class, it doesn't need to be a function inside a class. So,you should define it outside your class and use it inside the class functions. after that you remove 'eslint-disable-next-line'.